src/AutoRest.CSharp.V3/Generation/Writers/XmlCodeWriterExtensions.cs

```diff
@@ -375,5 +376,16 @@ private static void ToDeserializeValueCall(this CodeWriter writer, XmlValueSeria
                     throw new NotSupportedException();
             }
         }
+
+        public static void WriteMethodDeserialization(this CodeWriter writer, XmlElementSerialization serialization,
+            ref string destination, string response, string document = "document")
+        {
+            writer.Line($"var {document:D} = {typeof(XDocument)}.Load({response:D}.ContentStream, LoadOptions.PreserveWhitespace);");
+            writer.ToDeserializeCall(
+                serialization,
+                w => w.Append($"{document:D}"),
```

Comment:
document seems to be declared twice here.

Reply:
Fixed.